doc/sphinx/practical-tools/coqide.rst

```diff
@@ -23,21 +23,39 @@ for CoqIDE.  Use `coqide --help` to see the list of command line options.
 
 .. _coqide_mainscreen:
 
-  .. image:: ../_static/coqide.png
-     :alt: CoqIDE main screen
+.. image:: ../_static/coqide.png
+   :alt: CoqIDE main screen
+
+..  Here is the code used in the screenshot:
+
+    Fixpoint power (x n : nat) {struct n} : nat :=
+      match n with
+      | O => 1
+      | S m => x * power x m
+      end.
+
+    Notation "x ^ n" := (power x n).
+
+    Theorem Fermat :
+      (forall x y z n : nat, x^n + y^n = z^n -> n <= 2).
+    Proof.
+      Induction n.
 
 The screenshot shows CoqIDE as the user is stepping through the file `Fermat.v`.
+
 A menu bar and a tool bar appear at the top of the window. The left-hand panel shows
 the current *script buffer*.  Each script buffer corresponds to a separate Coq process.
 The upper right panel is the *proof panel*, which shows the goals to be proven.
 
-The lower right panel has three tabs: the *Messages
-panel*, which shows messages produced by commands and tactics; the *Errors panel*,
-which shows errors detected when running in :ref:`async mode <asyncmode>` and the
-*Jobs panel,* which shows information on the worker processes used by async mode.
-The contents of the right-hand panels are specific to the currently-displayed script.
+The lower right panel has three tabs:
+
+- the *Messages panel* shows messages produced by commands and tactics;
+- the *Errors panel* shows errors detected when running in :ref:`async mode <asyncmode>`;
```

Comment:
Could you mention the Warnings tab, too?

Reply:
Yes. Is it still related to async mode? What should be the wording please?

Reply:
How about "- the *Warnings panel* shows warnings detected when running in :ref:`async mode <asyncmode>`;"